requests/packages/__init__.py

```diff
@@ -34,9 +34,3 @@
 except ImportError:
     import chardet
     sys.modules['%s.chardet' % __name__] = chardet
-
-try:
-    from . import idna
-except ImportError:
-    import idna
-    sys.modules['%s.idna' % __name__] = idna
```

Comment:
So, here's the crux of the issue: I'm not sure we can remove this.

@sigmavirus24, we added these blocks to help tolerate the possibility of downstream repackagers unvendoring us. Do we still believe we need this?